content/blog/query-kubernetes/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Introducing Pulumi Query
+authors: ["joe-duffy"]
+tags: ["Pulumi-News", "Kubernetes"]
+meta_desc: "Today we announced Pulumi Pulumi Query, an SDK for programmatically querying cloud resources."
+date: "2019-11-14"
+meta_image: "pulumi-crosswalk-k8s.png"
+---
+
+It is too hard to answer simple questions about Kubernetes resources. Questions like:
```

Comment:
We could use a much softer and more inviting intro here.

Reply:
We often need answers to simple questions about Kubernetes resources.